src/main/java/org/zalando/nakadi/filters/LoggingFilter.java

```diff
@@ -0,0 +1,45 @@
+package org.zalando.nakadi.filters;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.stereotype.Component;
+import org.springframework.web.filter.OncePerRequestFilter;
+
+import javax.servlet.FilterChain;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+import java.security.Principal;
+import java.util.Optional;
+
+@Component
+public class LoggingFilter extends OncePerRequestFilter {
```

Comment:
Does it work without additional registration?
Because in WebConfig.java we register other filters we have and define the order in which the filters will be invoked https://github.com/zalando/nakadi/blob/master/src/main/java/org/zalando/nakadi/config/WebConfig.java#L54
Do you think it will make sense to add LoggingFilter there also?